<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Salvar</class>
 <widget class="QDialog" name="Salvar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Salvar Dados</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../imgs/imgs.qrc">
    <normaloff>:/imgs/save_all.png</normaloff>:/imgs/save_all.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Selecione os nomes dos arquivos de saída.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="chb_real_niveis">
         <property name="text">
          <string>Planta Real - Níveis</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chb_real_erro_sc">
         <property name="text">
          <string>Planta Real - Erro / Sin. Cont.</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chb_sim_niveis">
         <property name="text">
          <string>Simulação - Níveis</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chb_sim_erro_sc">
         <property name="text">
          <string>Simulação - Erro / Sin. Cont.</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="end_real_niveis">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="end_real_erro_sc">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="end_sim_niveis">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="end_sim_erro_sc">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QToolButton" name="selec_real_niveis">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../imgs/imgs.qrc">
           <normaloff>:/imgs/folder_blue.png</normaloff>:/imgs/folder_blue.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="selec_real_erro_sc">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../imgs/imgs.qrc">
           <normaloff>:/imgs/folder_blue.png</normaloff>:/imgs/folder_blue.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="selec_sim_niveis">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../imgs/imgs.qrc">
           <normaloff>:/imgs/folder_blue.png</normaloff>:/imgs/folder_blue.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="selec_sim_erro_sc">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../imgs/imgs.qrc">
           <normaloff>:/imgs/folder_blue.png</normaloff>:/imgs/folder_blue.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../imgs/imgs.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Salvar</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Salvar</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chb_real_niveis</sender>
   <signal>toggled(bool)</signal>
   <receiver>selec_real_niveis</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>580</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chb_real_erro_sc</sender>
   <signal>toggled(bool)</signal>
   <receiver>selec_real_erro_sc</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>580</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chb_sim_niveis</sender>
   <signal>toggled(bool)</signal>
   <receiver>selec_sim_niveis</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>580</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chb_sim_erro_sc</sender>
   <signal>toggled(bool)</signal>
   <receiver>selec_sim_erro_sc</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>580</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
